akka-cluster/src/multi-jvm/scala/akka/cluster/MultiNodeClusterSpec.scala

```diff
@@ -51,6 +51,7 @@ object MultiNodeClusterSpec {
 
       sharding {
         retry-interval = 200ms
+        waiting-for-state-timeout = 200ms
```

Comment:
Isn't that rather something to configure closer to the tests where it matters than in the global config?

Reply:
The test in question wasn't testing this, it just failed as the shard coordinator didn't become active quick enough due to the initial Read of the state being ignored. I think any test that uses ddata sharding could fail for the same reason so I added it here so they all pick it up.

Reply:
It looks to ignore reads: https://github.com/akka/akka/blob/master/akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala#L1248

Reply:
Ah, I was thinking of the Get (that generates the Read). Ok, that's correct. Then the Get will timeout after 5 seconds and I can understand that we want to retry.